scripts/typings/test_typings.js

```diff
@@ -0,0 +1,78 @@
+var spawnSync = require('child_process').spawnSync;
+var path = require('path');
+var fs = require('fs');
+
+// Suite variables:
+var passSuite = true;
+var errors = '';
+
+// Test functions:
+
+/**
+ * Runs transpile checks against the failed typed file and should show an error
+ * for each line from the start to the ending line.
+ *
+ * @param file The typescript file producing type errors
+ * @param start The starting line in the file where the first error occurs
+ * @param end The ending line number in which the final error occurs
+ */
+var testFailures = function(file, start, end) {
+  var spawned = spawnSync('node', ['node_modules/typescript/bin/tsc',
+      path.resolve('scripts/typings', file)]);
+  var output = spawned.output.toString();
+  var lines = output.split('\n');
+  var pos = 0;
+  var pass = 0;
+
+  for (var currentError = start; currentError <= end; currentError++) {
+    var line = lines[pos];
+    if (line.indexOf(file + '(' + currentError) >= 0) {
+      pass++;
```

Comment:
I was using `pass` previously and I am no longer using it. Removing it.

`pos` is still used to advance through the lines.